lib/inets/examples/server_root/conf/ssl.conf

```diff
@@ -1,66 +1,55 @@
-ErrorLog logs/error_log_8088
-TransferLog logs/access_log_8088
-ErrorDiskLog logs/error_disk_log_8088
-ErrorDiskLogSize 200000 10
-TransferDiskLog logs/access_disk_log_8088
-TransferDiskLogSize 200000 10
-MaxClients 150
-DocumentRoot /var/tmp/server_root/htdocs
-DirectoryIndex index.html welcome.html
-DefaultType text/plain
-Alias /icons/ /var/tmp/server_root/icons/
-Alias /pics/ /var/tmp/server_root/icons/
-ScriptAlias /cgi-bin/ /var/tmp/server_root/cgi-bin/
-ScriptAlias /htbin/ /var/tmp/server_root/cgi-bin/
-ErlScriptAlias /cgi-bin/erl httpd_example io
-EvalScriptAlias /eval httpd_example io
-SSLCertificateFile /var/tmp/server_root/ssl/ssl_server.pem
-SSLCertificateKeyFile /var/tmp/server_root/ssl/ssl_server.pem
-SSLVerifyClient 0
-#Script HEAD /cgi-bin/printenv.sh
-#Action image/gif /cgi-bin/printenv.sh
-
-<Directory /var/tmp/server_root/htdocs/open>
-AuthName Open Area
-AuthUserFile /var/tmp/server_root/auth/passwd
-AuthGroupFile /var/tmp/server_root/auth/group
-require user one Aladdin
-</Directory>
-
-<Directory /var/tmp/server_root/htdocs/secret>
-AuthName Secret Area
-AuthUserFile /var/tmp/server_root/auth/passwd
-AuthGroupFile /var/tmp/server_root/auth/group
-require group group1 group2
-</Directory>
-
-<Directory /var/tmp/server_root/htdocs/secret/top_secret>
-AuthName Top Secret Area
-AuthUserFile /var/tmp/server_root/auth/passwd
-AuthGroupFile /var/tmp/server_root/auth/group
-require group group3
-</Directory>
-
-<Directory /var/tmp/server_root/htdocs/mnesia_open>
-AuthName Open Area
-AuthMnesiaDB On
-require user one Aladdin
-</Directory>
-
-<Directory /var/tmp/server_root/htdocs/mnesia_secret>
-AuthName Secret Area
-AuthMnesiaDB On
-require group group1 group2
-</Directory>
-
-<Directory /var/tmp/server_root/htdocs/mnesia_secret/top_secret>
-AuthName Top Secret Area
-AuthMnesiaDB On
-require group group3
-</Directory>
+[{port, 8088},
+ {server_name, "your.server.net"},
+ {socket_type, {essl,[]}},
+ {modules, [mod_alias,mod_auth,mod_esi,mod_actions,mod_cgi,mod_dir,mod_get,
+            mod_head,mod_log,mod_disk_log]},
+ {server_admin, "jocke@erix.ericsson.se"},
+ {server_root, "/var/tmp/server_root"},
+ {error_log, "logs/error_log_8088"},
+ {transfer_log, "logs/access_log_8088"},
+ {error_disk_log, "logs/error_disk_log_8088"},
+ {error_disk_log_size, {200000,10}},
+ {transfer_disk_log, "logs/access_disk_log_8088"},
+ {transfer_disk_log_size, {200000,10}},
+ {max_clients, 150},
+ {document_root, "/var/tmp/server_root/htdocs"},
+ {directory_index, ["index.html","welcome.html"]},
+ {default_type, "text/plain"},
+ {alias, {"/icons/", "/var/tmp/server_root/icons/"}},
+ {alias, {"/pics/", "/var/tmp/server_root/icons/"}},
+ {script_alias, {"/cgi-bin/", "/var/tmp/server_root/cgi-bin/"}},
+ {script_alias, {"/htbin/", "/var/tmp/server_root/cgi-bin/"}},
+ {erl_script_alias, {"/cgi-bin/erl", [httpd_example]}},
+ {eval_script_alias, {"/eval",[httpd_example]}},
+ {ssl_certificate_file, "/var/tmp/server_root/ssl/ssl_server.pem"},
```

Comment:
This will no longer work, that was dropped.  ssl options should be {ssl, [{certfile, "/var/tmp/server_root/ssl/ssl_server_cert.pem"},{keyfile, "/var/tmp/server_root/ssl/ssl_key.pem"}, {verify, verify_none}]}. To match the old options in this example